plugins/modules/eos_ospfv2.py

```diff
@@ -0,0 +1,1458 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for eos_ospfv2
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "network",
+}
+
+DOCUMENTATION = """
+---
+module: eos_ospfv2
+version_added: 2.10
+short_description: Configures and manages attributes of ospf on Arista EOS platforms.
+description: This module configures and manages the attributes of ospf on Arista EOS platforms.
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
```

Comment:
Can we add what vEOS version this module was tested with and the supported connections?